ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Ds3Cli.java

```diff
@@ -193,8 +197,17 @@ private CliCommand getCommandExecutor() {
             case GET_PHYSICAL_PLACEMENT: {
                 return new GetPhysicalPlacement(this.ds3Provider, this.fileUtils);
             }
-            case GET_SERVICE:
+            case GET_TAPE_FAILURE: {
+                return new GetTapeFailure(this.ds3Provider, this.fileUtils);
+            }
+            case DELETE_TAPE_FAILURE: {
+                return new DeleteTapeFailure(this.ds3Provider, this.fileUtils);
+            }
+            case GET_SERVICE: {
+                return new GetService(this.ds3Provider, this.fileUtils);
+            }
             default: {
+                System.err.println("Command not recognized");
```

Comment:
I would log this.  And most likely we should end up changing this to return an exception, but we should do that work in a different pull request.